web/sites/default/config/views.view.media.yml

```diff
@@ -4362,7 +4357,7 @@ display:
           exposed: true
           expose:
             operator_id: uid_op
-            label: Author
+            label: Name
```

Comment:
This line would revert the fix for "Author" filter name.

Reply:
I must have missed the merge of your story into this branch. Fixed and rebuilding.